src/core_modules/capture-core/components/WidgetProfile/WidgetProfile.component.js

```diff
@@ -48,31 +49,22 @@ export const WidgetProfile = ({ teiId, programId }: Props) => {
     const loading = programsLoading || trackedEntityInstancesLoading;
     const error = programsError || trackedEntityInstancesError;
 
+    const listAttributes = useListAttributes(programsData, trackedEntityInstancesData);
```

Comment:
NIT: Maybe `useClientAttributesWithSubvalues` and `clientAttributesWithSubvalues` (or something that doesn't imply that this has something to do with displayInList)